services/private_ip_manager/src/main/resources/application.properties

```diff
@@ -1,13 +1,15 @@
+server.port=9004
 #Redis configuration
-spring.redis.host=localhost
-spring.redis.port=6380
+#spring.redis.host=localhost
+#spring.redis.port=6380
 #apache.kafka.address=172.17.0.1:9092
 #server.port=9000
 #Ignite configuration
+#ignite.host=192.168.137.1
```

Comment:
Switching back and forward on different host ip is kind of headache. Let us create a new set of application.properties to be more developer friendly :-) when you get a chance.

Reply:
Sure